akka-bbb-apps/src/main/scala/org/bigbluebutton/core/running/MeetingActor.scala

```diff
@@ -173,6 +173,7 @@ class MeetingActor(
 
   //Insert meeting into the database
   MeetingDAO.insert(liveMeeting.props, liveMeeting.clientSettings)
+  PluginModel.persistPluginsForClient(liveMeeting.plugins, props.meetingProp.intId)
```

Comment:
I suggest to move this line to inside `MeetingDAO.insert` to avoid race conditions (trying to insert plugins before the meeting was inserted). 
You could call `MeetingDAO.insert(liveMeeting.props, liveMeeting.clientSettings, liveMeeting.plugins)`